master.sh

```diff
@@ -0,0 +1,11 @@
+for index in $(seq 0 2); do
```

Comment:
I don't see why this is a shell script? Couldn't the loop be done in python?

Reply:
This is just copied from the install instructions - don't want to change what they've given us if it works